dashboard/test/ui/browsers.json

```diff
@@ -38,13 +44,24 @@
   {
     "name": "iPad",
     "platformName": "iOS",
-    "appium:platformVersion": "14.5",
+    "appium:platformVersion": "16.2",
     "browserName": "safari",
     "appium:deviceName": "iPad Simulator",
     "appium:mobile": true,
     "appium:automationName": "XCUITest",
     "sauce:options": {
       "deviceOrientation": "LANDSCAPE"
     }
-  }
+  },
+  {
+    "name": "Android",
```

Comment:
We would need to filter these out via `@no_mobile` and such tags otherwise we run Android tests on features that we know fail on `iOS`.

Reply:
We need to add:

```
"appium:mobile": true,
```